app/views/challenges/implementation_status.html.haml

```diff
@@ -0,0 +1,22 @@
+#challenge-48in24-page
+  %section.top-section
+    .lg-container
+      .flex.xl:flex-row.flex-col.xl:items-start.items-center
+        .xl:mr-auto{ class: 'max-w-[860px]' }
+          %table
```

Comment:
It's not particularly pretty and it definitely won't be pretty once we have 48 exercises to render